### DISCUSSION TITLE
adding an extra PWM input

### DISCUSSION THREAD

**Discussion by Neillives** [ExpressLRS/ExpressLRS#2447] [Q&A]
Hi I'm flying a T1 VTOL with ELRS receiver RP1
I want to add a pan servo however all the PWM outputs on my FC are all being used, what would be the best way to achieve this ? keeping with Elrs 2.4 ?
I looked at changing the receiver to a elrs one that has physical PWM inputs but i would then need one that goes past 10 channels i think 

Did see Betafpv have a Elrs 14ch PWM receiver that works CRSF and Sbus 

Any advise would be great thanks 

NI

**Comment by r1000ru** [comment]
You can use one uart TX as sbus Output on your FC. It possible on Ardupilot and Inav.
